graylog2-server/src/main/java/org/graylog2/shared/bindings/ValidatorModule.java

```diff
@@ -24,6 +24,6 @@
 public class ValidatorModule extends AbstractModule {
     @Override
     protected void configure() {
-        bind(Validator.class).toProvider(ValidatorProvider.class);
+        bind(Validator.class).toProvider(ValidatorProvider.class).asEagerSingleton();
```

Comment:
It would be simpler to directly bind to the instance `Validation.buildDefaultValidatorFactory().getValidator()` instead of using a singleton provider which basically returns a static field. This way we could get rid of the `Provider<Validator>`.

https://github.com/google/guice/wiki/InstanceBindings